wrong_bit_types.csv

```diff
@@ -0,0 +1,5 @@
+StructWithBitsWithoutOF,3001,DataType
```

Comment:
if this file is located (like github shows it) in the main folder please move it to tests